Attendee.js

```diff
@@ -0,0 +1,39 @@
+// The Attendee class can be utilized to keep certain 
+// information of the attendee, a class for the user 
+// needed to be created in addition to the session class. 
+// The test for this JS class file utilizes Jest.IO
+class Attendee {
+    constructor(sessionId, screenName, timeLastPolled) {
+    /** @private {string} */
+        this.sessionId_ = sessionId;
+    /** @private {string} */
+        this.screenName_ = screenName;
+    /** @private {string} */
+        this.timeLastPolled_ = timeLastPolled;  
+        
+    } // end of constructor
+    
+    getSessionId() {
+        return this.sessionId;
+    }
+
+    getScreenName() {
+        return this.screenName;
+    }
+
+    getTimeLastPolled() {
+        return this.timeLastPolled;
+    }
+/**
+   * @param {Attendee} obj An attendee object must be passed so that the function 
```

Comment:
The param is not of type Attendee. Its of type Object. It is a javascript object that is basically the JSON that is returned from the server. It basically looks like this on the caller side:

{
"blah" : "blah",
"blah" : "blah"
}

This function is converting an arbitrary javascript object into an Attendee object. The code is correct but the documentation is inaccurate/outdated.